### PR TITLE
Specify group in BaseKtlintCheckTask

### LINKED ISSUE #350: Set verification group on ktlint tasks
Currently tasks are group-less, which means they go on to a "other" catch-all. Would be nice if these could go in the verification group. Happy to send a PR.

### PR DESCRIPTION
**Pull request by ZacSweers**
Resolves #350

### CODE REVIEW

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt [hunk 1]
```diff
@@ -32,6 +32,10 @@ import org.jlleitschuh.gradle.ktlint.reporter.ReporterType
 abstract class BaseKtlintCheckTask(
     private val objectFactory: ObjectFactory
 ) : SourceTask() {
+    
+    init {
+      group = "ktlint"
```
Comment (ZacSweers) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt:37]:
```suggestion
        group = "ktlint"
```

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt [hunk 2]
```diff
@@ -32,6 +32,10 @@ import org.jlleitschuh.gradle.ktlint.reporter.ReporterType
 abstract class BaseKtlintCheckTask(
     private val objectFactory: ObjectFactory
 ) : SourceTask() {
+
```
Comment (ZacSweers) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/BaseKtlintCheckTask.kt:35]:
```suggestion

```

**Comment by ZacSweers** [comment]
CI failure looks like a flake in caching

```
Post job cleanup.
/bin/tar -cz -f /home/runner/work/_temp/51f76031-aac8-4246-9090-9c73c6066ebb/cache.tgz -C /home/runner/.gradle/caches .
[warning]connect ECONNREFUSED 13.107.42.16:443
events.js:187
      throw er; // Unhandled 'error' event
      ^

Error: ESPIPE: invalid seek, read
Emitted 'error' event on ReadStream instance at:
    at internal/fs/streams.js:167:12
    at FSReqCallback.wrapper [as oncomplete] (fs.js:470:5) {
  errno: -29,
  code: 'ESPIPE',
  syscall: 'read'
}
```

**Comment by Tapchicoma** [comment]
@ZacSweers travis for now is a main CI for this project and we are trying out how Github actions are working for this project, so it is fine if it fails.

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt [hunk 3]
```diff
@@ -356,7 +356,7 @@ private fun Project.createAndroidVariantMetaKtlintCheckTask(
     variantName: String,
     multiplatformTargetName: String? = null
 ): TaskProvider<Task> = registerTask(variantName.androidVariantMetaCheckTaskName(multiplatformTargetName)) {
-    group = VERIFICATION_GROUP
+    group = KTLINT_GROUP
```
Comment (Tapchicoma) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt:359]: Initial discussion was to change groupless tasks to have "Verification" group, though in this PR you've changed tasks that already had "Verification" group to have "ktlint" group. Why so?

Comment (ZacSweers) [reply plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt:359]: Ah I was going off of the later comment o make them ktlint due to the volume and noise it causes

**Pull request closed without merging**